app/models/reject.rb

```diff
@@ -1,5 +1,5 @@
 class Reject < ActiveRecord::Base
   belongs_to :user
   has_many :users
-  validates :user_id, uniquness: true, scope: :reject_id, message: "already rejected"
+  validates_uniqueness_of :user_id, scope: :reject_id, message: "already rejected"
```

Comment:
Line is too long. [82/80]<br>Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.